Dockerfile

```diff
@@ -23,15 +23,10 @@ WORKDIR /home
 COPY --from=builder /home/jpo-ode-svcs/src/main/resources/application.properties /home
 COPY --from=builder /home/jpo-ode-svcs/src/main/resources/logback.xml /home
 COPY --from=builder /home/jpo-ode-svcs/target/jpo-ode-svcs.jar /home
+COPY startup_jpoode.sh /home
 
-ENTRYPOINT ["java", \
-	"-Djava.rmi.server.hostname=$DOCKER_HOST_IP", \
-	"-Dcom.sun.management.jmxremote.port=9090", \
-	"-Dcom.sun.management.jmxremote.rmi.port=9090", \
-	"-Dcom.sun.management.jmxremote", \
-	"-Dcom.sun.management.jmxremote.local.only=true", \
-	"-Dcom.sun.management.jmxremote.authenticate=false", \
-	"-Dcom.sun.management.jmxremote.ssl=false", \
-	"-Dlogback.configurationFile=/home/logback.xml", \
-	"-jar", \
-	"/home/jpo-ode-svcs.jar"]
+RUN apk add openssh
```

Comment:
Do any additional ports need to be exposed for the SSH functionality to work? If so, these should be added to the 'ode' service in the docker compose files.

Reply:
Yeah I can expose this to work locally. I haven't tested it locally so I can't really vouch for a way to do that properly. This is definitely a K8s solution.

Reply:
I have thought about this for longer and I actually don't think I would recommend running this locally. This is really an inconvenient way to manage SSH access to the ODE system if you have the ability to use a standard host machine's SSH server. Mounting a directory from that host machine to the Dockerized ODE is definitely the best way to go about this. I have included the K8s documentation for this rationale and how to deploy it in K8s where we do expose the port.

Reply:
Sounds good!